db/migrate/20221219111209_change_invoice_subscription_source.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class ChangeInvoiceSubscriptionSource < ActiveRecord::Migration[7.0]
+  def up
+    remove_column :invoice_subscriptions, :source
```

Comment:
Rails/BulkChangeTable: You can use `change_table :invoice_subscriptions, bulk: true` to combine alter queries.